src/components/select.tsx

```diff
@@ -9,7 +8,7 @@ import { Spinner } from './spinner';
 export type Option<T extends string> = {
   value: T;
   label: string;
-  iconURL?: URL;
```

Comment:
Would this be more idiomatic React as just `Icon?: React.ReactElement`? And then `<Icon/>` instead of `{icon()}`?

Definitely don't feel strongly about it, but I had to scan back and forth a couple of times to work out how `getIcon` was plumbed through into `renderTag`.

Reply:
This comment was supposed to be on the added line not the removed line. Makes sense in side by side diff but confusing in the main PR view!